content/en/docs/contribute/generate-ref-docs/kubernetes-components.md

```diff
@@ -118,134 +118,79 @@ need to work with someone who can set the label and milestone for you.
 
 The `update-imported-docs` tool performs these steps:
 
-1. Clone the `kubernetes/kubernetes` repository.
-1. Run several scripts under `kubernetes/kubernetes/hack`. These scripts
-   generate Markdown files and place the files under `kubernetes/kubernetes/docs`.
-1. Copy the generated Markdown files to a local clone of the `kubernetes/website`
-   repository under `kubernetes/website/docs/reference/generated`.
-1. Clone the `kubernetes/federation` repository.
-1. Run several scripts under `kubernetes/federation/hack`. These scripts
-   generate Markdown files and place the files under `kubernetes/federation/docs`.
-1. Copy the generated Markdown files to a local clone of the `kubernetes/website`
-   repository under `kubernetes/website/docs/reference/generated`.
-
-After the Markdown files are in your local clone of the `kubernetes/website`
+1. Clones the related repositories specified in a configuration file. For the
+   purpose of generating reference docs, the repositories that are cloned by
+   default are `kubernetes-incubator/reference-docs` and `kubernetes/federation`.
+1. Runs commands under the cloned repositories to prepare the docs generator and
+   then generates the Markdown files.
+1. Copies the generated Markdown files to a local clone of the `kubernetes/website`
+   repository under locations specified in the configuration file.
+
+When the Markdown files are in your local clone of the `kubernetes/website`
 repository, you can submit them in a
 [pull request](https://kubernetes.io/docs/home/contribute/create-pull-request/)
 to `kubernetes/website`.
 
-## Setting the branch
+## Customizing the config file
 
-Open `<web-base>/update-imported-docs/config.yaml` for editing.
-
-Set the value of `branch` to the Kubernetes release that you want to document.
-For example, if you want to generate docs for the Kubernetes 1.9 release,
-set `branch` to `release-1.9`.
+Open `<web-base>/update-imported-docs/reference.yaml` for editing.
+Do not change the content for the `generate-command` entry unless you know what
```

Comment:
"...unless you know what you are doing" is somewhat vague. Instead, how about "Do not change the content for the `generate-command` entry unless you understand what it is doing and need to change the specified release branch."

Reply:
Done.